npm/src/upload/index.ts

```diff
@@ -48,8 +48,12 @@ export class FileUploader {
     uploadFilesInfo: FileUploadInfo
   ) => {
     window.addEventListener("beforeunload", pageUnloadAction)
-    scheduleTokenRefresh(this.keycloak, `${this.uploadUrl}/cookies`)
-
+    const cookiesUrl = `${this.uploadUrl}/cookies`
+    scheduleTokenRefresh(this.keycloak, cookiesUrl)
+    await fetch(cookiesUrl, {
```

Comment:
Does this first `fetch` still take a while if the Lambda is cold? And does that cause any problems if the user starts the upload too quickly?

I'm wondering whether any of the new users who might be uploading a few small files every day might get used to the workflow and end up clicking things really quickly.

Reply:
This happens after the start upload button is pressed. So they press submit and the call to the `/cookies` endpoint starts and then they see the upload progress bar. The fetch does take a while with a cold start so the user will see the progress bar sat on 0% for ~10s (worse case) and then it will start to upload. 
Once the fetch has returned, then the client side processing starts and the upload progress bar progresses.